### DISCUSSION TITLE
Selecting "Liked models only" doesn't work.

### DISCUSSION THREAD

**Discussion by lololh** [BlafKing/sd-civitai-browser-plus#256] [Ideas]
Ok, i just discovered this extension a few days ago and i tried it to download the models i had selected with the "like" functionality on civitai. 
Here is how i work. As often as i can, i choose the models i'd like to use by clicking on the thumb on civitai... it would be great if they implement a function "to test", but that's not the problem here... When i have time, i go to my account with favorites models and i download the models i selected the days before...
Thanks to your extension i can do it very quickly but... it doesn't work... 
When I check "liked models only", nothing happens ... I don't have any change on the models displayed. I can't say if it worked before as i'm trying it only for a few days... 
Tell me if i'm doing something wrong.
I have my Personal CivitAI API key in the settings tab.
THank your for your job. Greats from France

**Comment by BlafKing** [comment]
Thanks for reporting!

It seems that the liked models filtering is currently broken in the CivitAI API, there is an existing issue on their github for it [here](https://github.com/civitai/civitai/issues/1110)
As this is an issue with the API, it's unfortunately not something that I can fix in the extension itself.

I'll turn this discussion into an issue and I'll leave it open until the issue is fixed 👍🏻

**Comment by BlafKing** [comment]
I'll close this discussion as I've made a bug report for it, you can find it [here](https://github.com/BlafKing/sd-civitai-browser-plus/issues/257)!

**Reply by lololh** [reply]
Ok thank you. I'll wait. Thanks for your work
